### ISSUE TITLE
Double tap doesn't quite zoom to the position double tapped on

### ISSUE THREAD

**Issue by cristan** [nos-digital/imagin#7]
On imagin 1.2.1:

Activity XML:
```
<?xml version="1.0" encoding="utf-8"?>
<androidx.constraintlayout.widget.ConstraintLayout
    xmlns:android="http://schemas.android.com/apk/res/andro
    xmlns:app="http://schemas.android.com/apk/res-auto"
    android:id="@+id/product_zoom_layout"
    android:layout_width="match_parent"
    android:layout_height="match_parent">
    <ImageView
        android:importantForAccessibility="no"
        android:id="@+id/product_image"
        android:layout_width="0dp"
        android:layout_height="0dp"
        app:layout_constraintBottom_toBottomOf="parent"
        app:layout_constraintEnd_toEndOf="parent"
        app:layout_constraintStart_toStartOf="parent"
        app:layout_constraintTop_toTopOf="parent"/>
</androidx.constraintlayout.widget.ConstraintLayout>
```

Imagin code:
```
Glide.with(this)
    .load("https://www.jqueryscript.net/images/Universal-Placeholder-Text-Lorem-Ipsum-Generator-getlorem.jpg")
    .into(binding.productImage)

Imagin.with(
    imageViewWrapper = binding.productZoomLayout,
    imageView = binding.productImage,
    maxZoom = 5f
).enableDoubleTapToZoom()
```

Double tapping results in zooming in in not quite the correct position. It works fine if you double tap in the center, but if you double tap outside of it, you are zoomed in somewhere else:
https://user-images.githubusercontent.com/490738/132707349-f8dce48e-118e-4c16-afd8-583f6125148c.mp4

**Comment by mennovogel** [comment]
Yes, I think this should be fixed. I've created a fix for it, but I don't have access to push my commits. @vuongp @TimGW or @jelleheemskerk can you give me access to push my commits so we can fix this?

**Comment by vuongp** [comment]
@mennovogel Done, you should have write access to the repo now.

**Comment by vuongp** [comment]
This should be improved with Menno his fix in 1.2.2

Thanks guys!
